lib/options.go

```diff
@@ -299,6 +299,10 @@ type Options struct {
 	// iteration is shorter than the specified value.
 	MinIterationDuration types.NullDuration `json:"minIterationDuration" envconfig:"K6_MIN_ITERATION_DURATION"`
 
+	// Cloud is the config for the cloud
+	// formally known as ext.loadimpact
+	Cloud json.RawMessage `json:"cloud,omitempty"`
```

Comment:
```suggestion
	Cloud json.RawMessage `json:"cloud,omitempty"`
```
Why not use the opportunity of this change for defining a strict struct instead of using `json.RawMessage`? Sorry, for asking only now.

Reply:
Agreed internally that this could increase the scope of the issue, but doesn't bring the value for original PRs purpose (introducing the `options.cloud`). So if we need to switch to the struct we can do that later.